docs/source/installation.rst

```diff
@@ -150,25 +150,15 @@ Step 3: Install ``torch`` and ``torchvision`` with CPU or GPU Support
 
                     .. code::
 
-                       pip install torch==1.8.1+cu111 torchvision==0.9.1+cu111 --find-links https://download.pytorch.org/whl/torch_stable.html
+                       pip install --extra-index-url https://download.pytorch.org/whl/cu111 ivadomed
 
                 .. tab:: CPU Support
 
                     If you plan to run ``ivadomed`` on CPU only, install PyTorch per instructions provided below for your specific operating system:
 
-                    .. tabs::
-
-                        .. tab:: Windows/Linux
-
-                            .. code::
-
-                               pip install torch==1.8.0+cpu torchvision==0.9.0+cpu --find-links https://download.pytorch.org/whl/torch_stable.html
-
-                        .. tab:: Mac
-
-                            .. code::
+                    .. code::
```

Comment:
TODO: make sure this renders right on RTD

Reply:
Seems okay: https://ivado-medical-imaging--1129.org.readthedocs.build/installation.html

<details>

![Screenshot 2022-05-17 at 16-41-23 Installation — ivadomed documentation](https://user-images.githubusercontent.com/987487/168906359-3f40d17b-9db6-4ae5-8231-43f7560cc365.png)
![Screenshot 2022-05-17 at 16-41-30 Installation — ivadomed documentation](https://user-images.githubusercontent.com/987487/168906364-c152c9af-99ce-41b4-afb5-b06c45e38ae2.png)

</details>

There's more tweaks that should be made here, probably. The entire 'Step 3' should be merged into Step 2 now.